constitution.md

```diff
@@ -91,6 +91,10 @@ The social media team is responsible for the outreach of the community, by tweet
 
 The engagement officers are responsible for coming up with new initiatives (projects, summer schools, outreach activities, etc.) and propose them to the community. They will also oversee ongoing initiatives and provide support for them to reach their goals. They will ensure that the initiative is known in the community, as well as outside the community together with the media team. As well as suggesting potential funding sources for the initiative, by engaging with the treasurer team.
 
+### IDEA (Inclusion, Diversity, Equity, and Accessibility) team 
+
+The IDEA's coordinators' primary objective is to establish an atmosphere where every individual feels appreciated, included, and empowered. They are your primary contact for any concerns, suggestions, or ideas related to these areas. If you encounter challenges—including conflicts, incidents, or acts of aggression do not hesitate to reach out the current [coordinators](/steering_committee.md#roles).
+
 <!-- ### Time in Office
 
 The Steering Committee will be in office for a period of two years, after which a new committee will be chosen. Roles are decided collectively at the start of a new term.
```

Comment:
Should we be uncommenting these parts btw? 🤔 
We made the announcement in Slack a while back, but I guess we never updated the Website to state this is how it will work.